.github/workflows/verify-pr-target.yml

```diff
@@ -1,14 +1,21 @@
 name: Verify pull request target
 
-on: [pull_request]
+on: [pull_request_target]
 
 jobs:
   check_pr_target:
     runs-on: ubuntu-latest
     steps:
+    - name: Send comment if wrong pull request target
+      if: github.base_ref == 'master'
```

Comment:
We'll have to remember to change this after reorganizing and renaming the branches.